html/editing/editing-0/spelling-and-grammar-checking/references/spelling-markers-011-ref.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html lang="en" class="reftest-wait">
+<meta charset="utf-8">
+<title>Reference file for spellcheck tests</title>
+<style>
+  div {
+    padding: 10px;
+    background: silver;
+  }
+</style>
+
+<p>The test passes if you see a spelling error marker in the word "quikc".</p>
```

Comment:
Can you add a comment that this ref is used as a mismatch, to help explain why the text here seems to contradict the following `spellcheck=false`?